src/main/java/org/cactoos/scalar/Folded.java

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
@bedward70 Please, look my review of `HighestOf` PR (#575). All comments are aplicable here too.